cirq/atoms/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cirq.atoms.atom_devices import (
+    AQuA,
```

Comment:
This specific device should be under a package in the same way that `cirq.google.Bristlecone` is.

Reply:
I named the class after a specific device but that is actually the general use atom device class (similar to xmondevice). I'll rename it to be more generic.